packages/core/src/Footer/About/About.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import useStyles from "../useStyles";
+
+import RichTypography from "../../RichTypography";
+
+function About({ variant = "caption", children, ...props }) {
```

Comment:
Use [`defaultProps`](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values)
```suggestion
function About({ children, variant, ...props }) {
```